apps/nextjs-example/package.json

```diff
@@ -30,6 +30,7 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-toast": "^1.1.5",
+    "@telegram-apps/sdk": "^2.0.0",
```

Comment:
In CAD we use '@telegram-apps/sdk-react' - is it a different package?

https://github.com/aptos-labs/create-aptos-dapp/blob/main/templates/clicker-game-tg-mini-app-template/package.json#L31

Reply:
It's the same underlying package (although I'm using v2 here)

'react-sdk' builds on top of 'sdk'

Reply:
didn't need full react support